xarrayutils/utils.py

```diff
@@ -61,7 +68,7 @@ def xr_linregress(a, b, dim='time', convert_to_dataset=True, dtype=None):
 
     Parameters
     ----------
-    a : {xr.DataArray}
+    a : {xr.DataArray} (does it have to be tho? my 'a' is vector)
```

Comment:
That is just the type of the input. The data can be 1 dimensional (which is what I assume you mean with 'vector') or n-dimensional. http://xarray.pydata.org/en/stable/generated/xarray.DataArray.html

Reply:
oops forgot I left that note sorry!